micro-rdk/src/common/robot.rs

```diff
@@ -612,6 +623,7 @@ impl LocalRobot {
                             status,
                         });
                     }
+                    _ => continue
```

Comment:
to cover the `ResourceType::Button` case since it doesn't `impl Status` anymore. This will also help for the `Switch` PR.